src/AppInstallerCLITests/Filesystem.cpp

```diff
@@ -72,9 +72,11 @@ TEST_CASE("VerifyIsSameVolume", "[filesystem]")
     std::filesystem::path path9 = L"a";
     std::filesystem::path path10 = L"b";
 
-    // Verify that a relative path points to the current volume.
     REQUIRE(IsSameVolume(path1, path2));
-    REQUIRE(IsSameVolume(path5, path7));
+    if (IsSameVolume(path5, path5))

```

Comment:
Is this if check intentional?

Reply:
This test is poorly considered, as it requires one to have a `D` drive, otherwise it will fail.  The underlying OS API fails if the path does not reference an extant volume, and our function treats that as a sort of `NaN` result and always returns false if either path results in it.  This means that `D:\path1` and `D:\path2` will result in `false` if `D` is not a volume.  As a minimal workaround to that, I attempted to determine if `D` exists by checking if equal paths point to the same volume.

The test would be better if it constructed the paths based on the existing volumes rather than hard coding specific paths.  But I was just trying to put a bandaid on the issue.